src/components/Guilds/common/Modal/index.tsx

```diff
@@ -0,0 +1,149 @@
+import styled from 'styled-components';
+import ReactDOM from 'react-dom';
+
+import { Button } from '../Button';
+import { FiX } from 'react-icons/fi';
+import { Heading } from '../Typography';
+
+export const Wrapper = styled.div`
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 700;
+  width: 40%;
+  min-width: 400px;
+  max-width: ${({ maxWidth }) => maxWidth}px;
+  outline: 0;
+
+  @media only screen and (max-width: 768px) {
+    max-width: 300px;
+    min-width: 300px;
+    width: 300px;
+  }
+`;
+
+export const Backdrop = styled.div`
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  background: rgba(0, 6, 41, 0.5);
+  z-index: 500;
+`;
+export const StyledModal = styled.div`
+  z-index: 100;
+  background: ${({ theme }) => theme.colors.background};
+  position: relative;
+  margin: auto;
+  border: 1px solid ${({ theme }) => theme.colors.muted};
+  border-radius: ${({ theme }) => theme.radii.curved};

```

Comment:
```suggestion
  border-radius: ${({ theme }) => theme.radii.curved2};
```
Should be 10px instead of 6. 
https://www.figma.com/file/cjLng0AbOKuDj9r5oZVDmR/Wireframe?node-id=1051%3A9774